Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -33,10 +33,14 @@ public partial class DatePicker<TValue> : BaseTextInput<IReadOnlyList<TValue>>,
     /// <inheritdoc/>
     public override async Task SetParametersAsync( ParameterView parameters )
     {
+        var datesUsed = parameters.TryGetValue( nameof( Dates ), out IEnumerable<TValue> paramDates );
```

Comment:
Isn't this always true?

Reply:
It is not. Blazor will only pass what is been explicitly defined as a parameter,

Reply:
I honestly didn't know about that. I thought every parameter was always populated with defaults in that collection. Guess I was wrong ^^